code/game/objects/structures/crates_lockers/crates/wooden.dm

```diff
@@ -15,4 +15,116 @@
 	new	/obj/item/reagent_containers/food/drinks/soda_cans/canned_laughter(src)
 	new /obj/item/pneumatic_cannon/pie(src)
 	new /obj/item/reagent_containers/food/snacks/pie/cream(src)
-	new /obj/item/storage/crayons(src)
\ No newline at end of file
+	new /obj/item/storage/crayons(src)
+
+/obj/structure/closet/crate/wooden/artifact/PopulateContents()
+	. = ..()
+	switch (pickweight(list("worldwar" = 3, "tommy" = 2, "gang" = 1, "rushn" = 2, "detective" = 1, "wiz_charge" = 2, "wiz_summon_forcewall" = 2, "wiz_blind" = 1, "wiz_sacredflame" = 1, "wiz_barnyard" = 1, "wiz_knock_smoke" = 1, "wiz_door" = 2, "wiz_fireball" = 1, "wiz_suit" = 2, "cutlass" = 2, "katana" = 1, "pharaoh" = 2, "knight" = 1, "bo" = 2, "hook" = 1, "spiders" = 2, "tiki" = 1)))
+		if("worldwar")
+			new /obj/item/clothing/suit/armor/vest/old(src)
+			new /obj/item/gun/ballistic/rifle/boltaction(src)
+			new /obj/item/grenade/syndieminibomb/concussion(src)
+			new /obj/item/clothing/head/helmet/old(src)
+
+		if("tommy")
+			new /obj/item/gun/ballistic/automatic/tommygun/no_mag(src)
+			new /obj/item/ammo_box/magazine/tommygunm45/empty(src)
+			for(var/i = 0, i < 4, ++i)
+				new /obj/item/ammo_casing/c45(src)
+			new /obj/item/clothing/head/fedora(src)
+
+		if("gang")
+			new /obj/item/gun/ballistic/automatic/mini_uzi(src)
```

Comment:
And for this one it might be worth considering one empty spare magazine and then maybe a flavor item like a spraycan, or a bandana.

Reply:
Still needs the spare magazine.

Reply:
Bling necklace too